### PR TITLE
Make edu-benefits-submit-success style common

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#3817

Replaced via `grep -rl 'education-benefits-submit-success' vets-website/* | xargs sed -i "" 's/education-benefits-submit-success/benefit-submit-success/g'`

### CODE REVIEW

File: src/js/burials/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ class ConfirmationPage extends React.Component {
     const { deathCertificate, transportationReceipts } = form.data;
 
     return (
-      <div className="edu-benefits-submit-success">
+      <div className="benefit-submit-success">
```
Comment (jbalboni) [inline src/js/burials/containers/ConfirmationPage.jsx]: Do we even need this class? It seems like we could make a common `page-title` style that does the centering and have the tests look for that.

Comment (cvalarida) [inline src/js/burials/containers/ConfirmationPage.jsx]: ☝️

Comment (cehsu) [inline src/js/burials/containers/ConfirmationPage.jsx]: Capital idea, @jbalboni. Changes made.

File: src/js/burials/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ class ConfirmationPage extends React.Component {
   }
 
   componentDidMount() {
-    focusElement('.burial-page-title');
+    focusElement('.page-title');
```
Comment (jbalboni) [inline src/js/burials/containers/ConfirmationPage.jsx]: Page title is a little generic. Maybe `confirmation-page-title`?

Comment (cehsu) [inline src/js/burials/containers/ConfirmationPage.jsx]: Thanks @jbalboni, change made.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**